gvsbuild/projects/expat.py

```diff
@@ -44,11 +44,11 @@ def post_install(self):
             # Fontconfig is looking for libexpat, not libexpatd
             bin_dir = Path(self.builder.gtk_dir) / "bin"
             self.builder.exec_msys(
-                ["mv", "libexpatd.dll", "libexpat.dll"],
```

Comment:
in gvsbuild we do not want libraries named with the d suffix

Reply:
@nacho Can you elaborate please? This PR makes it possible to get a successful debug build.

Reply:
I mean that probably what we should fix is fontconfig

Reply:
Hello @nacho. I think fontconfig is OK. A better solution is to simply remove the post_install code that does the copying. I believe it to be redundant/obsolete. I'll try that next pass.

Reply:
So removing the `post_install` works as expected for a debug build of `font_config`. Shall I send a new PR?

```
(.venv) PS R:\src\gtk\test\gvsbuild> git diff
diff --git a/gvsbuild/projects/expat.py b/gvsbuild/projects/expat.py
index 750fc70a..42799bf0 100644
--- a/gvsbuild/projects/expat.py
+++ b/gvsbuild/projects/expat.py
@@ -39,16 +39,3 @@ class Expat(Tarball, CmakeProject):
         CmakeProject.build(self, use_ninja=True)
         self.install(r".\COPYING share\doc\expat")

-    def post_install(self):
-        if self.builder.opts.configuration == "debug":
-            # Fontconfig is looking for libexpat, not libexpatd
-            bin_dir = Path(self.builder.gtk_dir) / "bin"
-            self.builder.exec_msys(
-                ["mv", "libexpatd.dll", "libexpat.dll"],
-                working_dir=bin_dir,
-            )
-            lib_dir = Path(self.builder.gtk_dir) / "lib"
-            self.builder.exec_msys(
-                ["mv", "libexpatd.lib", "libexpat.lib"],
-                working_dir=lib_dir,
-            )
(.venv) PS R:\src\gtk\test\gvsbuild>   
```

Reply:
I added that code in #989 to attempt to fix another issue. It may not have been the right fix, but we need to make sure fontconfig is linking to the right dll.

Reply:
Hello @danyeaw. Yes, this works fine. I've just rerun a debug build of librsvg with zero problems.

```
R:\src\gtk\test\gvsbuild>gvsbuild build --verbose --capture-out --ninja-opts -j=2  --build-dir r:\src\gtk\test\build --msys-dir r:/apps/msys64 --log-single --configuration debug librsvg
...
Building project librsvg (2.56.2)

Project(s) built:
    cargo                     (32.867 s)
    ninja                     (1.515 s)
    meson                     (4.343 s)
    pkgconf                   (7.320 s)
    cmake                     (4.606 s)
    zlib                      (3.781 s)
    libpng                    (10.018 s)
    freetype                  (11.497 s)
    gperf                     (5.767 s)
    expat                     (11.835 s)
    fontconfig                (18.315 s)
    win-iconv                 (4.129 s)
    gettext                   (87.422 s)
    libffi                    (23.399 s)
    pcre2                     (11.527 s)
    glib                      (129.976 s)
    pixman                    (9.328 s)
    cairo                     (72.498 s)
    harfbuzz                  (104.121 s)
    fribidi                   (9.909 s)
    pango                     (24.848 s)
    nasm                      (1.518 s)
    libjpeg-turbo             (26.526 s)
    libtiff-4                 (30.750 s)
    gdk-pixbuf                (24.844 s)
    libxml2                   (14.103 s)
    librsvg                   (77.515 s)

```